sysutils/tmux/Makefile

```diff
@@ -0,0 +1,67 @@
+# Created by: Wen Heping <wenheping@gmail.com>
+
+PORTNAME=	tmux
+PORTVERSION=	3.2
+CATEGORIES=	sysutils
+MASTER_SITES=	https://github.com/tmux/tmux/releases/download/${PORTVERSION}/
+DISTFILES=	${DISTNAME}${EXTRACT_SUFX}
+
+MAINTAINER=	kraileth@elderlinux.org
+COMMENT=	Terminal Multiplexer
+
+LICENSE=			iscl gpl2
+LICENSE_COMB=			multi
+LICENSE_FILE_ISCL=		${WRKSRC}/COPYING
+LICENSE_FILE_GPL2=		${WRKSRC_bash}/COPYING
+LICENSE_DISTFILES_ISCL=		${DISTNAME}${EXTRACT_SUFX}
+LICENSE_DISTFILES_GPLv2=	${DISTFILE_bash}
+
+USES=		ncurses pkgconfig
+
+USE_GITHUB=	nodefault
+GH_TUPLE=	imomaliev:tmux-bash-completion:f5d5323:bash
+
+GNU_CONFIGURE=	yes
+CONFIGURE_ARGS=	--enable-utempter \
+		--sysconfdir=${PREFIX}/etc
+
+PLIST_FILES=	bin/tmux \
+		etc/bash_completion.d/tmux \
+		man/man1/tmux.1.gz
+
+PORTDOCS=	CHANGES README
+PORTEXAMPLES=	*
+
+OPTIONS_DEFINE=	DOCS EXAMPLES LIBEVENT_STATIC UTF8PROC
+
+LIBEVENT_STATIC_DESC=	Build with static libevent
+UTF8PROC_DESC=		Build with utf8proc support
+
+LIBEVENT_STATIC_BUILD_DEPENDS=		${LOCALBASE}/lib/libevent.a:devel/libevent
+LIBEVENT_STATIC_LIB_DEPENDS_OFF=	libevent.so:devel/libevent
+LIBEVENT_STATIC_VARS=			LESTATIC=${LOCALBASE}/lib/libevent.a
+
+UTF8PROC_LIB_DEPENDS=		libutf8proc.so:textproc/utf8proc
+UTF8PROC_CONFIGURE_ENABLE=	utf8proc
+
+post-patch:
+	@${REINPLACE_CMD} -e 's|/etc/tmux.conf|${PREFIX}/etc/tmux.conf|g' ${WRKSRC}/CHANGES \
+		${WRKSRC}/tmux.h
+
+post-configure-LIBEVENT_STATIC-on:
+	${REINPLACE_CMD} -e '/LIBS/s|-levent[^[:blank:]]*|${LESTATIC}|' \
+		${WRKSRC}/${MAKEFILE}
+
+post-install:
+	@${MKDIR} ${PREFIX}/etc/bash_completion.d
+	${INSTALL_DATA} ${WRKSRC_bash}/completions/tmux ${PREFIX}/etc/bash_completion.d/tmux
+
+post-install-DOCS-on:
+	@${MKDIR} ${FAKE_DESTDIR}${DOCSDIR}
+	(cd ${WRKSRC} && ${INSTALL_DATA} ${PORTDOCS} ${FAKE_DESTDIR}${DOCSDIR})
+
+post-install-EXAMPLES-on:
+	@${MKDIR} ${EXAMPLESDIR}
+	${INSTALL_DATA} ${WRKSRC}/example* ${EXAMPLESDIR}
```

Comment:
I think you're going to need to add ${FAKE_DESTDIR} in front of the ${EXAMPLESDIR} entries here.  

The general pattern is that do-install/post-install and related are going to inject ${FAKE_DESTDIR} into prefix for you, but the variants like post-install-[option name]-on: will not.